src/symbol.cr

```diff
@@ -29,11 +38,21 @@ struct Symbol
     end
   end
 
+  # Returns the name or string corresponding to sym.
```

Comment:
Well, this form doesn't return it. It also doesn't do different things, so the "or" could be a bit misleading. How about "Writes the symbols name to the passed io"?